woocommerce/woocommerce
# Enhanced `pnpm clean`

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This adds `.wireit` directories to `pnpm clean` and fixes the ordering so `pnpm store prune` works as it should.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Run `pnpm install && pnpm build` to make sure all of the files are populated.
2. Run `pnpm clean` from the repository root. Observe that `.wireit` and `node_modules` directories are deleted. Note that the clean runs an install after and will quickly re-populate `node_modules` directories.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
